src/openrct2/ride/RideStringIds.h

```diff
@@ -0,0 +1,208 @@
+    STR_RIDE_NAME_REVERSER_ROLLER_COASTER = 67,
+    STR_RIDE_NAME_REVERSE_FREEFALL_COASTER = 44,
+    STR_RIDE_NAME_SIDE_FRICTION_ROLLER_COASTER = 55,
+    STR_RIDE_NAME_SINGLE_RAIL_ROLLER_COASTER = 94,
+    STR_RIDE_NAME_SPIRAL_ROLLER_COASTER = 2,
+    STR_RIDE_NAME_STAND_UP_ROLLER_COASTER = 3,
+    STR_RIDE_NAME_STEEPLECHASE = 12,
+    STR_RIDE_NAME_SUSPENDED_SWINGING_COASTER = 4,
+    STR_RIDE_NAME_TWISTER_ROLLER_COASTER = 53,
+    STR_RIDE_NAME_VERTICAL_DROP_ROLLER_COASTER = 46,
+    STR_RIDE_NAME_VIRGINIA_REEL = 61,
+    STR_RIDE_NAME_WATER_COASTER = 76,
+    STR_RIDE_NAME_WILD_MOUSE = 56,
+    STR_RIDE_NAME_WOODEN_ROLLER_COASTER = 54,
+    STR_RIDE_NAME_WOODEN_WILD_MOUSE = 11,
+    STR_RIDE_DESCRIPTION_AIR_POWERED_VERTICAL_COASTER = 587,
+    STR_RIDE_DESCRIPTION_ALPINE_COASTER = 605,
+    STR_RIDE_DESCRIPTION_BOBSLEIGH_COASTER = 525,
+    STR_RIDE_DESCRIPTION_CLASSIC_MINI_COASTER = 6119,
+    STR_RIDE_DESCRIPTION_CLASSIC_STAND_UP_ROLLER_COASTER = 607,
+    STR_RIDE_DESCRIPTION_CLASSIC_WOODEN_ROLLER_COASTER = 606,
+    STR_RIDE_DESCRIPTION_COMPACT_INVERTED_COASTER = 585,
+    STR_RIDE_DESCRIPTION_CORKSCREW_ROLLER_COASTER = 531,
+    STR_RIDE_DESCRIPTION_FLYING_ROLLER_COASTER = 569,
+    STR_RIDE_DESCRIPTION_GIGA_COASTER = 580,
+    STR_RIDE_DESCRIPTION_HEARTLINE_TWISTER_COASTER = 578,
+    STR_RIDE_DESCRIPTION_HYBRID_COASTER = 603,
+    STR_RIDE_DESCRIPTION_HYPERCOASTER = 6113,
+    STR_RIDE_DESCRIPTION_HYPER_TWISTER = 563,
+    STR_RIDE_DESCRIPTION_INVERTED_HAIRPIN_COASTER = 588,
+    STR_RIDE_DESCRIPTION_INVERTED_IMPULSE_COASTER = 598,
+    STR_RIDE_DESCRIPTION_INVERTED_ROLLER_COASTER = 515,
+    STR_RIDE_DESCRIPTION_JUNIOR_ROLLER_COASTER = 516,
+    STR_RIDE_DESCRIPTION_LAY_DOWN_ROLLER_COASTER = 574,
+    STR_RIDE_DESCRIPTION_LIM_LAUNCHED_ROLLER_COASTER = 602,
+    STR_RIDE_DESCRIPTION_LOOPING_ROLLER_COASTER = 527,
+    STR_RIDE_DESCRIPTION_MINE_RIDE = 600,
+    STR_RIDE_DESCRIPTION_MINE_TRAIN_COASTER = 529,
+    STR_RIDE_DESCRIPTION_MINI_ROLLER_COASTER = 599,
+    STR_RIDE_DESCRIPTION_MINI_SUSPENDED_COASTER = 519,
+    STR_RIDE_DESCRIPTION_MULTI_DIMENSION_ROLLER_COASTER = 567,
+    STR_RIDE_DESCRIPTION_REVERSER_ROLLER_COASTER = 577,
+    STR_RIDE_DESCRIPTION_REVERSE_FREEFALL_COASTER = 554,
+    STR_RIDE_DESCRIPTION_SIDE_FRICTION_ROLLER_COASTER = 565,
+    STR_RIDE_DESCRIPTION_SINGLE_RAIL_ROLLER_COASTER = 604,
+    STR_RIDE_DESCRIPTION_SPIRAL_ROLLER_COASTER = 512,
+    STR_RIDE_DESCRIPTION_STAND_UP_ROLLER_COASTER = 513,
+    STR_RIDE_DESCRIPTION_STEEPLECHASE = 522,
+    STR_RIDE_DESCRIPTION_SUSPENDED_SWINGING_COASTER = 514,
+    STR_RIDE_DESCRIPTION_TWISTER_ROLLER_COASTER = 6116,
+    STR_RIDE_DESCRIPTION_VERTICAL_DROP_ROLLER_COASTER = 556,
+    STR_RIDE_DESCRIPTION_VIRGINIA_REEL = 571,
+    STR_RIDE_DESCRIPTION_WATER_COASTER = 586,
+    STR_RIDE_DESCRIPTION_WILD_MOUSE = 566,
+    STR_RIDE_DESCRIPTION_WOODEN_ROLLER_COASTER = 564,
+    STR_RIDE_DESCRIPTION_WOODEN_WILD_MOUSE = 521,
+    STR_SPINNING_WILD_MOUSE_GROUP = 6162,
```

Comment:
This name are a leftover from Ride Groups. Seems like a good moment to change to `STR_RIDE_NAME_SPINNING_WILD_MOUSE` and `STR_RIDE_DESCRIPTION_SPINNING_WILD_MOUSE`.